src/hotspot/share/utilities/preserveErrno.hpp

```diff
@@ -0,0 +1,16 @@
+#include <errno.h>
+
+
+// Preserve errno across a range of calls
```

Comment:
You also need to expose the preserved value so it can be logged.

Reply:
+1